x-pack/plugins/reporting/server/lib/enqueue_job.ts

```diff
@@ -12,63 +12,52 @@ import { LevelLogger } from './';
 import { Report } from './store';
 import type { ReportingRequestHandlerContext } from '../types';
 
-export type EnqueueJobFn = (
+export async function enqueueJob(
+  reporting: ReportingCore,
+  request: KibanaRequest,
+  context: ReportingRequestHandlerContext,
+  user: ReportingUser,
   exportTypeId: string,
   jobParams: BaseParams,
-  user: ReportingUser,
-  context: ReportingRequestHandlerContext,
-  request: KibanaRequest
-) => Promise<Report>;
-
-export function enqueueJobFactory(
-  reporting: ReportingCore,
   parentLogger: LevelLogger
-): EnqueueJobFn {
+): Promise<Report> {
   const logger = parentLogger.clone(['createJob']);
-  return async function enqueueJob(
```

Comment:
It's probably easier to review changes like this with the feature in Github that lets you ignore whitespace changes: add `?w=1` to the github URL.